generators/base/command.ts

```diff
@@ -38,6 +39,17 @@ const command = {
       type: Boolean,
       scope: 'generator',
     },
+    blueprint: {
+      description: 'DEPRECATED: Specify a generator blueprint to use for the sub generators',
```

Comment:
@mshima This option was already deprecated in v8, so we can delete it now in v9

Reply:
It’s deprecated since v7 or v6 before I’ve joined the team 😄 
We should do in another PR.